### PR TITLE
Layout base & main components

### PR DESCRIPTION
**Pull request by germanf**

### CODE REVIEW

File: app/components/Footer/index.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <Wrapper>
+      <section>
+        <FormattedMessage {...messages.licenseMessage} />
+      </section>
+      <section>
+        <FormattedMessage
+          {...messages.authorMessage}
+          values={{
+            author: <A href="https://www.omniwallet.org/">OmniWallet</A>,
```
Comment (achamely) [inline app/components/Footer/index.jsx]: we should update the author to 'The Omni Foundation', http://www.omnilayer.org/

File: app/components/HeaderMessage/messages.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * This contains all the text for the HeaderMessage component.
+ */
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  header: {
+    id: 'app.components.HeaderMessage.header',
+    defaultMessage: '{note} The OmniExplorer.info domain will transition to the Omni Foundation at the end of Feb 2018, where a new and improved block explorer will take over.',
```
Comment (achamely) [inline app/components/HeaderMessage/messages.js]: we should update this to 'Welcome to the new Omniexplorer.info'

File: app/components/ServiceBlock/index.jsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+const ContainerSummaryItem3 = styled.div`
+  background-color: #727CB6;
+  padding: 6px;
+  margin: 0 6px;
+  font-size: 0.9rem
+`;
+
+const SummaryItem1 = () => (<ContainerSummaryItem1 className="text-white"><span className="d-block lead" style={{ fontSize: '0.9rem' }}>TODAY'S OMNI PRICE</span><span className="lead">0.008234</span></ContainerSummaryItem1>);
+const SummaryItem2 = () => (<ContainerSummaryItem2 className="text-white"><span className="d-block lead" style={{ fontSize: '0.9rem' }}>TODAY'S TRANSACTIONS</span><span className="lead">8975</span></ContainerSummaryItem2>);
+const SummaryItem3 = () => (<ContainerSummaryItem3 className="text-white"><span className="d-block lead" style={{ fontSize: '0.9rem' }}>OMNI PROPERTIES</span><span className="lead">342 <small>(+377 test)</small></span></ContainerSummaryItem3>);
```
Comment (achamely) [inline app/components/ServiceBlock/index.jsx]: Note: These numbers will need to come from api calls

File: app/components/Transaction/index.jsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+const IMG = styled.img`
+  width: 48px;
+  height: 48px;
+  margin-right: 7px;
+`;
+
+class Transaction extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
+  render() {
+    return (
```
Comment (achamely) [inline app/components/Transaction/index.jsx]: note: these will need to come from api calls as well

File: app/components/ServiceBlock/index.jsx (hunk 3 above)
Comment (germanf) [reply app/components/ServiceBlock/index.jsx]: Yes, that data is mocked until to consume the API.

File: app/components/Transaction/index.jsx (hunk 4 above)
Comment (germanf) [reply app/components/Transaction/index.jsx]: Yes, that data is mocked until to consume the API.

**Comment by germanf** [comment]
The messages fixes are ready (notification & footer).
The mocked data are used until to have connected the API.

**Pull request merged**